pandas/tests/test_categorical.py

```diff
@@ -3682,6 +3682,20 @@ def f():
         exp = Series([True, True, True, False])
         res = (cat == cat)
         tm.assert_series_equal(res, exp)
+        
```

Comment:
these need to go in pandas/tests/indexing/test_indexing (and put in the Categorical section)